src/fideslang/validation.py

```diff
@@ -44,6 +45,25 @@ def sort_list_objects_by_name(values: List) -> List:
     return values
 
 
+def unique_items_in_list(values: List) -> List:
+    """
+    Verify that the names of each item in the provided list are unique.
```

Comment:
super nit: could make it a _bit_ more explicit, either in the docstring or the function name, that we're looing specifically at the `name` attribute on each element in the provided list. it feels like an OK assumption to make in the validator, but the more clearly it can be called out (that we're assuming and leveraging the `name` attribute specifically) the better.

```suggestion
    Verify that the `name` attributes of each item in the provided list are unique.
```

or something like that?

Reply:
this is a great suggestion! Not a nit at all 🙂 Thank you!